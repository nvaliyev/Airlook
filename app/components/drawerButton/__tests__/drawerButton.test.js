import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Burger from '..';
describe('Burgerbutton', () => {
    test(' Burger button snapshot', () => {
        const renderer = new ShallowRenderer();
        const snapshot = renderer.render(<Burger/>);
        expect(snapshot).toMatchSnapshot();
    });
});


