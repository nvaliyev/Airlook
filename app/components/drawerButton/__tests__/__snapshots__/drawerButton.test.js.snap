// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Burgerbutton  Burger button snapshot 1`] = `
<View
  style={
    Object {
      "flexDirection": "row",
      "marginLeft": 15,
    }
  }
>
  <TouchableOpacity
    activeOpacity={0.2}
    onPress={[Function]}
  >
    <Icon
      allowFontScaling={false}
      color="#F7F9FC"
      name="navicon"
      size={28}
    />
  </TouchableOpacity>
</View>
`;
